.idea/misc.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-    <component name="MavenProjectsManager">
-        <option name="originalFiles">
-            <list>
-                <option value="$PROJECT_DIR$/pom.xml" />
-            </list>
-        </option>
-    </component>
-    <component name="ProjectRootManager" version="2" languageLevel="JDK_12" project-jdk-name="12" project-jdk-type="JavaSDK">
-        <output url="file://$PROJECT_DIR$/classes" />
-    </component>
-</project>
```

Comment:
When you delete this file the IntelliJ configuration break. MMM @juanpa097 how we could fix this?

Reply:
I think that as long as you save a copy of it and just paste it after the commit, everything should be working fine.

Reply:
I think I do not commit this files. I ignore them and if intellij brakes just re import the project as maven. But we can sync on this offline.